Auth0/WebAuthError.swift

```diff
@@ -6,6 +6,7 @@ public struct WebAuthError: Auth0Error {
 
     enum Code: Equatable {
         case noBundleIdentifier
+        case transactionActiveAlready
```

Comment:
There are tests for `WebAuthError`, those should be updated as well: https://github.com/auth0/Auth0.swift/blob/master/Auth0Tests/WebAuthErrorSpec.swift

Reply:
Thanks for pointing it out, included tests for it now.

Reply:
There's also a test for the error code: https://github.com/auth0/Auth0.swift/blob/master/Auth0Tests/WebAuthErrorSpec.swift#L27